src/contentful/categories.ts

```diff
@@ -0,0 +1,18 @@
+import { ContentfulCategoryType } from '@/types/categoriesType'
+import contentfulClient from './contentful'
+import { parseThumbnail } from './helpers/parseThumbnail'
+
+export async function parseContentfulCategories() {
```

Comment:
I think it might be better to rename this to `getContentfulCategories` or `getCategories`